api/functions/delete_datasets.py

```diff
@@ -0,0 +1,7 @@
+from typing import Dict
+from functions_framework import Context
```

Comment:
Check our requirements.txt file to see if functions_framework is still there, because I believe I may have removed it in a previous PR

Reply:
Hmm I think it is still being used in `process_dataset.py` tho - also if `Context` from `functions_framework` is removed, what type hint do we use for the `context` parameter to the cloud functions?

Reply:
I think that when I looked it up, I saw it was mostly used for emulating cloud functions locally, and so I thought oh we don't do that yet let me scale back the requirements. But yeah because it's used for those files already, would you be able to add it back to requirements.txt if it's missing and pin it with a compatible version?

Reply:
Yes I can do that!